flexget/components/notify/notifiers/gotify.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import base64
+import datetime
+import re
+
+from flexget import plugin
+from flexget.event import event
+from flexget.config_schema import one_or_more
+from flexget.plugin import PluginWarning
+from flexget.utils.requests import Session as RequestSession, TimedLimiter
+from requests.exceptions import RequestException
+
+plugin_name = 'gotify'
+log = logging.getLogger(plugin_name)
+
+requests = RequestSession(max_retries=3)
+
+gotify_url_pattern = {
+    'type': 'string',
+    'pattern': r'^http(s?)\:\/\/.*\/message$',
+    'error_pattern': 'Gotify URL must begin with http(s) and end with `/message`',
+}
+
+class GotifyNotifier(object):
+    """
+    Example::
+
+    notify:
+      entries:
+        via:
+          - gotify:
+              url: <GOTIFY_SERVER_URL>
+              token: <GOTIFY_TOKEN>
+              priority: <PRIORITY>
+    Configuration parameters are also supported from entries (eg. through set).
+    """
+    schema = {
+        'type': 'object',
+        'properties': {
+            'url': one_or_more(gotify_url_pattern),
```

Comment:
so you can send to multiple urls via one call? how will that work?

Reply:
Yeah that's a bad usage of that, definitely didn't intend that. I think I want to just do this instead?

```
    schema = {
        'type': 'object',
        'properties': {
            'url': gotify_url_pattern,
```

Reply:
https://github.com/Flexget/Flexget/pull/2533/commits/3e5e7ada328e332dd5f4f5937ccf8e52a9d130f9